app/src/main/java/io/github/jbellis/brokk/ContextManager.java

```diff
@@ -1013,61 +1013,9 @@ public void addVirtualFragment(VirtualFragment fragment) {
         pushContext(currentLiveCtx -> currentLiveCtx.addVirtualFragment(fragment));
     }
 
-    /**
-     * Update the Build fragment based on structured success/failure. Always clears previous BUILD_LOG fragments. Only
-     * adds a new fragment when the build failed.
-     */
-    @Override
-    public void updateBuildFragment(boolean success, String buildOutput) {
```

Comment:
Seems this is now unused in IContextManager too, which we could remove. That and `getProcessedBuildOutput`

Reply:
good catch, removed